kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/kube_apiserver_metrics.py

```diff
@@ -170,8 +170,18 @@ def submit_as_gauge_and_monotonic_count(self, metric_suffix, metric, scraper_con
                 _tags.append('{}:{}'.format(label_name, label_value))
             # submit raw metric
             self.gauge(metric_name, sample[self.SAMPLE_VALUE], _tags)
-            # submit rate metric
-            self.monotonic_count(metric_name + '.count', sample[self.SAMPLE_VALUE], _tags)
+            if monotonic_count:
+                # submit rate metric
+                self.monotonic_count(metric_name + '.count', sample[self.SAMPLE_VALUE], _tags)
+
+    def aggregator_unavailable_apiservice(self, metric, scraper_config):
+        """
+        This function replaces the tag "name" by "apiservice_name".
+        It assumes that every sample is tagged with `name`.
+        """
+        for sample in metric.samples:
+            sample[self.SAMPLE_LABELS]["apiservice_name"] = sample[self.SAMPLE_LABELS].pop("name")
+        self.submit_as_gauge_and_monotonic_count('.aggregator_unavailable_apiservice', metric, scraper_config, False)
```

Comment:
More of a general question, why does changing the tag from `name` to `apiservice_name` affect if the metric is submitted as a monotonic count? Wouldn't this be a breaking change since `.aggregator_unavailable_apiservice.count` will no longer be a metric?

Also think this is more of a nit, but the name of the function `submit_as_gauge_and_monotonic_count` seems a bit misleading if you can disable submitting the monotonic count.

Reply:
Hi @yzhan289 

> Wouldn't this be a breaking change since .aggregator_unavailable_apiservice.count will no longer be a metric?

We are introducing this metric in the current release, so for now it is not a breaking change because it has not been released yet.

Reply:
I renamed the metric function as `submit_metric`. Also we never had `.aggregator_unavailable_apiservice.count`. For now we only keep the gauge.
Should we also have the count ?

Reply:
No it should be fine, I originally thought that `*.aggregator_unavailable_apiservice.count` was already a metric since I didn't know this metric is introduced in the current release.